packages/frontend/package-lock.json

```diff
@@ -0,0 +1,266 @@
+{
```

Comment:
This was created with an older version of npm, the new npm workspaces manages everything in the root lockfile.  Can you delete this lockfile and rerun `npm install` at the root directory?